### PR TITLE
MCOL-5218: Fix mcs-loadbrm service start while reisntall.

### PR DESCRIPTION
**Pull request by mariadb-AlanMologorsky**
Removed requests external dependency to keep working with preinstalled python3. Add fallback to detect primary if CMAPI is unreachable. Fix various minor python code styling.

### CODE REVIEW

File: oam/install_scripts/mcs-loadbrm.py.in [hunk 1]
```diff
@@ -69,170 +216,153 @@ if __name__ == '__main__':
...
             cmd = 'systemctl start mcs-storagemanager'
-            retcode = subprocess.call(cmd, shell=True)
-            if retcode != 0:
-                print('Failed to start storagemanager. \
-{} exits with {}.'.format(cmd, retcode))
+            try:
+                subprocess.check_call(cmd, shell=True)
+            except subprocess.CalledProcessError as exc:
+                print(
+                    f'Failed to start storagemanager. {cmd} exits with',
```
Comment (drrtuy) [inline oam/install_scripts/mcs-loadbrm.py.in]: This format is no go IMHO. I didn't get how didn't it failed but the minimal python version we support doesn't support f'' strings formatting.

File: oam/install_scripts/mcs-loadbrm.py.in [hunk 2]
```diff
@@ -1,43 +1,183 @@
...
+        except HTTPError as exc:
+            print(
+                'Something goes wrong while requesting primary status ',
+                'through api.',
+                f'Got response code "{exc.code}" with reason "{exc.reason}".'
+            )
+        except URLError:
+            print(
+                'CMAPI became unavailable while trying',
+                'to request primary status.'
```
Comment (drrtuy) [inline oam/install_scripts/mcs-loadbrm.py.in:179]: I suggest you to discuss the texts with a native.

**Review by drrtuy — CHANGES_REQUESTED** [review]

File: oam/install_scripts/mcs-loadbrm.py.in (hunk 1 above)
Comment (mariadb-LeonidFedorov) [reply oam/install_scripts/mcs-loadbrm.py.in]: Can you please clarify, which is has minimal Python without f strings. Centos 7 has python 3.6.8, and f strings are included

Comment (mariadb-LeonidFedorov) [reply oam/install_scripts/mcs-loadbrm.py.in]: prove
➜  xbench git:(feature/columnstore-2) ✗ sudo docker run  -it centos:7         
[root@9e6328da92a6 /]# yum install python3                                                                                                                                                    
Loaded plugins: fastestmirror, ovl                                                             
Determining fastest mirrors                                                                                                                                                                   
 * base: mirror.hostnet.nl                                                                                                                                                                    
 * extras: centos.mirror.triple-it.nl                                                                                                                                                         
 * updates: centos.mirror.triple-it.nl                                                                                                                                                        
base                                                                                                                                                                   | 3.6 kB  00:00:00     
extras                                                                                                                                                                 | 2.9 kB  00:00:00     
updates                                                                                                                                                                | 2.9 kB  00:00:00     
(1/4): base/7/x86_64/group_gz                                                                                                                                          | 153 kB  00:00:00     
(2/4): extras/7/x86_64/primary_db                                                                                                                                      | 250 kB  00:00:00     
(3/4): base/7/x86_64/primary_db                                                                                                                                        | 6.1 MB  00:00:00     
(4/4): updates/7/x86_64/primary_db                                                                                                                                     |  17 MB  00:00:00     
Resolving Dependencies                                                                         
--> Running transaction check
---> Package python3.x86_64 0:3.6.8-18.el7 will be installed
--> Processing Dependency: python3-libs(x86-64) = 3.6.8-18.el7 for package: python3-3.6.8-18.el7.x86_64
--> Processing Dependency: python3-setuptools for package: python3-3.6.8-18.el7.x86_64
--> Processing Dependency: python3-pip for package: python3-3.6.8-18.el7.x86_64
--> Processing Dependency: libpython3.6m.so.1.0()(64bit) for package: python3-3.6.8-18.el7.x86_64                                                                                             
--> Running transaction check                                                                                                                                                                 
---> Package python3-libs.x86_64 0:3.6.8-18.el7 will be installed                                                                                                                             
--> Processing Dependency: libtirpc.so.1()(64bit) for package: python3-libs-3.6.8-18.el7.x86_64                                                                                               
---> Package python3-pip.noarch 0:9.0.3-8.el7 will be installed                                                                                                                               
---> Package python3-setuptools.noarch 0:39.2.0-10.el7 will be installed                                                                                                                      
--> Running transaction check                                                                                                                                                                 
---> Package libtirpc.x86_64 0:0.2.4-0.16.el7 will be installed                                                                                                                               
--> Finished Dependency Resolution                                                                                                                                                            
                                                                                                                                                                                              
Dependencies Resolved
                                               
==============================================================================================================================================================================================
 Package                                             Arch                                    Version                                           Repository                                Size
==============================================================================================================================================================================================
Installing:                                                                                                                                                                                   
 python3                                             x86_64                                  3.6.8-18.el7

Comment (mariadb-AlanMologorsky) [reply oam/install_scripts/mcs-loadbrm.py.in]: Removed f-strings formatting.

**Review by drrtuy — APPROVED** [review]

**Pull request merged**